paparazzi-gradle-plugin/src/test/projects/verify-resources-java/gradle.properties

```diff
@@ -0,0 +1,2 @@
+VERSION_NAME=0.5.0-SNAPSHOT
```

Comment:
Probably not needed, right? Accidental copy/pasta?

Reply:
It "needs" it because I copied the test fixture stuff from Wire.  Can clean up later.